portafly/src/i18n/locales/en/audience/account/account.yml

```diff
@@ -0,0 +1,67 @@
+---
+# Accounts > Account overview
+
+page_title: Accounts
```

Comment:
@dfennessy The page is about one specific Account, so maybe it should be singular?